rapid7_insight_agent/plugin.spec.yaml

```
@@ -0,0 +1,185 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: rapid7_insight_agent
+title: Rapid7 Insight Agent
+description: Take actions on an Insight Agent
+version: 1.0.0
+vendor: rapid7
+support: community
```

This is rapid7 integration

```suggestion
support: rapid7
```